cmd/storagenode-updater/main.go

```diff
@@ -194,25 +207,30 @@ func update(ctx context.Context, binPath, serviceName string, renameBinary renam
 			}
 			log.Println("finished downloading", downloadURL, "to", tempArchive.Name())
 
-			err = renameBinary(currentVersion)
+			tempBinPath := filepath.Join(os.TempDir(), serviceName)
+			err = unpackBinary(ctx, tempArchive.Name(), tempBinPath)
 			if err != nil {
 				return errs.Wrap(err)
 			}
 
-			err = unpackBinary(ctx, tempArchive.Name(), binPath)
+			downloadedVersion, err := binaryVersion(tempBinPath)
 			if err != nil {
 				return errs.Wrap(err)
```

Comment:
do we need some special error type here?
smth like ```var Error = errs.Class("satellite console error")```
and 
```return errs.Wrap(Error.Wrap(err))```

Reply:
this is just simple cli tool so we don't have too many custom errors here. `errs.Wrap()` is used mainly for having stacktraces.